doc/Programming-guide/Key-platform-features/Data-collection/Flume-log-appender/index.md

```diff
@@ -0,0 +1,424 @@
+---
+layout: page
+title: Flume log appender
+permalink: /:path/
+nav: /:path/Programming-guide/Key-platform-features/Data-collection/Flume-log-appender
+sort_idx: 50
+---
+
+{% assign root_url = page.url | split: '/'%}
+{% capture root_url  %} /{{root_url[1]}}/{{root_url[2]}}/{% endcapture %}
+
+* TOC
+{:toc}
+
+The Flume log appender encapsulates received logs into Flume events and sends these events to external Flume sources via Avro RPC.
+
+# Creating Flume log appender in Admin UI
+
+The easiest way to create a Flume log appender for your application is by using Admin UI.
+
+To create a log appender which will be integrated with Hadoop or Spark, do the following:
+
+1. In the <b>Log appenders</b> window, click <b>Add log appender</b>.
+2. Enter the log appender name and description, select the minimum and maximum supported log schema version, and select necessary log metadata fields.
+3. Set the log appender type to <i>Flume</i>.
+4. Fill in the Flume log appender configuration form. <br/>
+NOTE:
```

Comment:
All notes should be formatted according to the Documentation contribution guide.